ui/src/cloud/actions/orgsettings.ts

```diff
@@ -0,0 +1,48 @@
+// API
+import {getOrgSettings as getOrgSettingsAJAX} from 'src/cloud/apis/orgsettings'
+
+// Constants
+import {FREE_ORG_HIDE_UPGRADE_SETTING} from 'src/cloud/constants'
+
+// Types
+import {GetState, OrgSetting} from 'src/types'
+
+// Selectors
+import {getOrg} from 'src/organizations/selectors'
+
+export enum ActionTypes {
+  SetOrgSettings = 'SET_ORG_SETTINGS',
+}
+
+export type Actions = SetOrgSettings
+
+export interface SetOrgSettings {
+  type: ActionTypes.SetOrgSettings
+  payload: {orgSettings: OrgSetting[]}
+}
+
+export const setOrgSettings = (settings: OrgSetting[] = []): SetOrgSettings => {
+  return {
+    type: ActionTypes.SetOrgSettings,
+    payload: {orgSettings: settings},
+  }
+}
+
+export const setFreeOrgSettings = (): SetOrgSettings => {
+  return {
+    type: ActionTypes.SetOrgSettings,
+    payload: {orgSettings: [FREE_ORG_HIDE_UPGRADE_SETTING]},
+  }
+}
+
+export const getOrgSettings = () => async (dispatch, getState: GetState) => {
```

Comment:
there's also a `creators` vs `thunks` concept to separate out direct reducer interaction and more complex "action of actions" patterns, but i think it's a bit overkill for this action set, just wanted to let you know about it
https://github.com/influxdata/influxdb/blob/master/ui/src/dashboards/actions/thunks.ts

Reply:
Noted.